app/models/notification.rb

```diff
@@ -172,4 +174,12 @@ def subject_number
   def display_thread?
     Octobox.include_comments? && subjectable? && subject.present? && user.display_comments?
   end
+
+  def github_client
+    if app_installation.present?
```

Comment:
I think we'll need to add `app_installation` to the eager loading over here: https://github.com/octobox/octobox/blob/master/app/services/download_service.rb#L67 to avoid n+1 sql queries when syncing notifications.